app/submitter/converter.py

```diff
@@ -73,8 +73,8 @@ def convert_answers(
         "survey_id": survey_id,
         "flushed": flushed,
         "submitted_at": submitted_at.isoformat(),
-        "collection": _build_collection(metadata),
-        "metadata": _build_metadata(metadata),
+        "collection": build_collection(metadata),
```

Comment:
you need to update doc strings for `convert_answers` method  since the payload structure has changed?

Reply:
Added now